main/res/values/strings.xml

```diff
@@ -140,6 +140,7 @@
     <string name="log_posting_image">Posting images…</string>
     <string name="log_posting_generic_trackable">Posting %1$s %2$d/%3$d</string>
     <string name="log_clear">Clear</string>
+    <string name="log_templates">Found it: Log template …</string>
```

Comment:
I would either name it `Found it (log template)` or remove the space between "template" and "…"

Reply:
The "template" part is the most important of the line, so that shouldn't be in brackets. I'll remove the space before the ellipsis.